napari/_qt/qt_main_window.py

```diff
@@ -157,7 +158,23 @@ def _add_file_menu(self):
         )
         open_folder.triggered.connect(self.qt_viewer._open_folder)
 
-        screenshot = QAction('Screenshot', self._qt_window)
+        save_selected_layers = QAction(
+            'Save selected layer(s)...', self._qt_window
+        )
+        save_selected_layers.setShortcut('Ctrl+S')
+        save_selected_layers.setStatusTip('Save selected layers')
+        save_selected_layers.triggered.connect(
+            lambda: self.qt_viewer._save_layers(selected=True)
+        )
+
+        save_all_layers = QAction('Save all layers...', self._qt_window)
+        save_all_layers.setShortcut('Ctrl+Shift+S')
+        save_all_layers.setStatusTip('Save all layers')
+        save_all_layers.triggered.connect(
+            lambda: self.qt_viewer._save_layers(selected=False)
+        )
+
+        screenshot = QAction('Save Screenshot', self._qt_window)
```

Comment:
When I do save screen shot I don't get the helpful status tip at top of dialog like i do for save all layers and save selected - see

![image](https://user-images.githubusercontent.com/6531703/80930965-d010cb00-8d6b-11ea-90b5-fd4c1c7ef63f.png)

vs.

![image](https://user-images.githubusercontent.com/6531703/80930968-d8690600-8d6b-11ea-8ca5-31163d0257eb.png)

Not sure what the difference is

Reply:
added caption to the save_screenshot method